client/src/components/modal-forms/ArchiveHiredParticipantForm.js

```diff
@@ -1,22 +1,34 @@
 import React from 'react';
 import { Box } from '@material-ui/core';
 import Grid from '@material-ui/core/Grid';
+import moment from 'moment';
```

Comment:
We use moment in a ton of projects, but it's no longer current. What are your thoughts of using a new date library in future projects? Do you have any favourites?

Reply:
I used dayjs .... that is lightweight.

Reply:
`date-fns` has been my replacement for moment